docusaurus/docs/guides/saving-forms.md

```diff
@@ -0,0 +1,46 @@
+---
+id: saving-forms
+title: Saving forms
+sidebar_label: Saving forms
+---
+
+`ngx-form-object` provides three hooks that can be utilized to add additional functionalities to a saving process. The hooks will be executed one after another in the following order: `beforeSave`, `save`, `afterSave`. Of this three methods only `save` method is <b>required for saving</b> the form.
```

Comment:
It doesn't say when those hooks will be executed. They will be executed once `save` method on a formStore instance is invoked.